packages/data-explorer/src/store/header/actions.ts

```diff
@@ -16,5 +39,24 @@ export default {
     })
     commit('setPackageTables', packageTables)
     return packageTables
+  },
+
+  fetchBreadcrumbs: async ({ commit, getters, rootState }: { commit: any, getters: any, rootState: any }) => {
```

Comment:
I would recommend handling the correct order here, else you have to have in (your own) memory that the order is reversed.

Seeing mutations (unshift, instead of push) and  the breadcrumbbar.vue depends on a reverse array

If you fetch it, and then reverse the array one time, then there is upstream no more troubles.

Reply:
still feels a little strange to have order different from other on screen, but i flipped it